yt/docs/README.md

```diff
@@ -450,6 +450,8 @@ For information about images usage, read the [YFM documentation](https://diplodo
    $ yfm -i ytsaurus/yt/docs/ -o ytsaurus-doc-output
    ```
 6. Navigate to the `ytsaurus-doc-output` folder and then go to the `ru` or `en` directory, depending on the documentation language you want to check. When opening the `index.html` file, the main page of the documentation will be shown.
+   
+   You can also run `bash build-and-serve.sh` in `ytsaurus-doc-output` folder to run an HTTP server locally. It will print URLs you can use to access the documentation.
```

Comment:
I would say:

You can also run the [build-and-serve.sh](https://github.com/ytsaurus/ytsaurus/blob/main/yt/docs/build-and-serve.sh) script in the `ytsaurus/yt/docs` folder to run an HTTP server locally. It will print URLs you can use to access the documentation.

```bash
$ bash build-and-serve.sh ytsaurus-doc-output

Checking YFM installed...
...
Starting HTTP server, open the links in your browser:

- http://localhost:8888/en (English)
- http://localhost:8888/ru (Russian)
```